XamlControlsGallery/ControlPages/ListViewPage.xaml

```diff
@@ -72,29 +72,91 @@
+                </Grid.RowDefinitions>
+                <Grid Height="Auto" Grid.Row="1" Margin="5" HorizontalAlignment="{x:Bind MsgAlignment}">
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="*"/>
+                    </Grid.ColumnDefinitions>
+                    <StackPanel Height="75" Width="350" Padding="10 0 0 10" Background="{x:Bind BgColor}" CornerRadius="4">
+                        <TextBlock Text="{x:Bind MsgText}" Padding="0 10 0 0" FontSize="20" Foreground="White"></TextBlock>
+                        <TextBlock Text="{x:Bind MsgDateTime}" Padding="0 0 0 10" FontSize="15" Foreground="White"></TextBlock>
+                    </StackPanel>
+                </Grid>
             </Grid>
         </DataTemplate>
+
+        <DataTemplate x:Key="ContactListViewTemplate2" x:DataType="local1:Contact">
+            <TextBlock Text="{x:Bind Name}" x:Phase="1" Margin="0,5,0,5"/>
+        </DataTemplate>
+        
     </Page.Resources>
     <StackPanel>
-        <local:ControlExample HeaderText="A Simple ListView with Selection support">
-            <ListView
+        
+        <local:ControlExample HeaderText="Basic ListView with Simple DataTemplate">
+            <StackPanel>
+                <TextBlock Margin="0 0 0 15">
+                    This is a basic ListView that has the full source code below. <LineBreak/>Other samples on this page display only the additional markup needed to customize a simple ListView like this one.
+                </TextBlock>
+                <ListView
+                x:Name="BaseExample"
+                ItemTemplate="{StaticResource ContactListViewTemplate2}"
+                BorderThickness="1"
+                BorderBrush="DimGray"
+                Width="350" 
+                Height="400"
+                HorizontalAlignment="Left"/>
+            </StackPanel>
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;ListView
+    x:Name="BaseExample"
+    ItemsSource="{x:Bind ContactsList}"
+    ItemTemplate="{StaticResource ContactListViewTemplate}"
+    BorderThickness="1"
+    BorderBrush="DimGray"
+    Width="350" 
+    Height="400"/&gt;
+                    
+ &lt;DataTemplate x:Key="ContactListViewTemplate" x:DataType="local1:Contact"&gt;
+     &lt;TextBlock Text="{x:Bind Name}" x:Phase="1" Margin="0,5,0,5"/&gt;
+&lt;/DataTemplate&gt;
+                    
+&lt;!-- In this example, the ListView's ItemTemplate property is bound to a data template (shown above)
+called ContactListViewTemplate, defined in a Page.Resources section. 
+                    
+The data template is defined to display a Contact object (class defined in C# code-behind), and the text 
+displayed is bound to the Contact object's Name attribute.
+                    
+The ItemsSource for the ListView is an ObservableCollection&lt;Contact&gt; called ContactsList and is
+defined in the C# code-behind. --&gt;
```

Comment:
In addition to this comment, might be nice to show the actual code. (We have other samples that show both markup and code-behind in this space.)